python/tvm/topi/arm_cpu/conv2d_alter_op.py

```diff
@@ -252,12 +254,20 @@ def _alter_conv2d_layout(attrs, inputs, tinfos, out_type):
 
         return relay.nn.conv2d(*inputs, **new_attrs)
 
-    if topi_tmpl == "conv2d_NCHWc.x86":
+    if topi_tmpl == "conv2d_NCHWc.arm_cpu":
```

Comment:
Do we want to also include the `conv2d_NCHWc.x86` topi template here?

Reply:
Ok that's what I had suspected; it sounds like cleaning it up would be what's best to avoid confusion later!

Reply:
Removed!